CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing Guidelines
+
+First of all, thanks for taking the time to contribute!
+
+Before starting to work on an issue, please go through the existing issues and pull requests to make sure no one is working on the same issue at the time.
+
+Once you select a task to work on, please mention that in the issue to let others to know you are working on that. If it's a new issue, please create an issue and do as same.
+
+## Setting up the development environment
+
+If you are not a member of the [@99xt](https://github.com/99xt) github orginzation, you should fork this repository, make changes in your own fork, and then submit a pull-request. 
```

Comment:
@rehrumesh 
Actually I got this copied from https://github.com/99xt/react-fileupload/blob/master/CONTRIBUTING.md

So, should we update all the places or is there any other good example that you can provide me